draft-ietf-core-href.md

```diff
@@ -191,15 +191,15 @@ The components are subject to the following constraints:
    that may not know the scheme's default port -->
 
 7. {:#c-path} A path consists of zero or more path segments.
-   A path must not consist of a single zero-length path segment, which
-   is considered equivalent to a path of zero path segments.
+   Note that a path of just a single zero-length path segment is allowed —
+   this is considered equivalent to a path of zero path segments by
```

Comment:
_"this is considered equivalent to a path of zero path segments by HTTP and CoAP"_

Does this mean that we need to add special rules for normalisation and comparison of CRIs with scheme = http(s) and coap(s)?

Reply:
Comparing CRIs can always give false negatives at the scheme level because CRIs perform no scheme level normalization. (We've considered prescribing or defining that the port would be elided for comparisons when default, maybe we want to do something similar here.)

Reply:
Maybe just noting the increased chance for false negatives then?